low-level/files/opt/seagate/sspl/setup/sspl_setup.py

```diff
@@ -87,6 +95,20 @@ def add_args(parser: str, cls: str, name: str):
         parsers.add_argument('args', nargs='*', default=[], help='args')
         parsers.set_defaults(command=cls)
 
+    @staticmethod
+    def dump_global_config(prvsnr_global_config_url):
+        Conf.load(global_config, prvsnr_global_config_url)
+        url_spec = urlparse(global_config_dump_url)
+        path = url_spec.path
+        store_loc = os.path.dirname(path)
+        if not os.path.exists(store_loc):
+            os.makedirs(store_loc)
+        with open(path, "w") as f:
+            f.write("")
```

Comment:
f.close() not recommended?

Reply:
Not required as we open it with context manager(with). This will take care of closing the file once write operation is done.